### ISSUE TITLE
Incorrect documentation - get_omni

### ISSUE THREAD

**Issue by Caseyb87** [spacepy/spacepy#446]
The _get_omni_  [documentation](https://spacepy.github.io/autosummary/spacepy.omni.get_omni.html#spacepy.omni.get_omni) says that 'OMNI2' is a valid option for the **dbase** argument. Using this string, however, results in an error.

### Minimal example to reproduce issue:

```
import spacepy.time as spt
import spacepy.omni as om
import datetime as dt

ticks=spt.tickrange(dt.datetime(2009,12,1,12),dt.datetime(2009,12,1,13), dt.timedelta(minutes=1))
interval_omni = om.get_omni(ticks, dbase='OMNI2')

```

### Error message/Traceback:

```
  File "C:\Users\ username \Anaconda3\lib\site-packages\spacepy\omni.py", line 218, in get_omni
    raise IOError('Specified dbase ({0}) must be specified in spacepy.config'.format(dbase))

OSError: Specified dbase (OMNI2) must be specified in spacepy.config
```

In get_omni.py, the following are listed as dbase options:

```
    dbase : str (optional)
        Select data source, options are 'QDhourly', 'OMNI2hourly', 'Mergedhourly'
        Note - Custom data sources can be specified in the spacepy config file
        as described in the module documentation.
```

I was hoping "OMNI2" would be one-min resolution, rather than interpolated hourly data, but if it is the hourly data then the documentation should say "OMNI2hourly".

**Comment by jtniehof** [comment]
Thanks for the report. The documentation was fixed in #241; the docs on the web are for the current release, not master. The [OMNI2](https://omniweb.gsfc.nasa.gov/html/overview.html) dataset is indeed hourly; the one-minute set is [high-resolution OMNI](https://omniweb.gsfc.nasa.gov/ow_min.html). The high-res OMNI is of course a much larger data set, so adding SpacePy handling of it would be a substantive effort. The current approach of downloading and processing it all in advance wouldn't be practicable.

**Comment by Caseyb87** [comment]
> 
> 
> Thanks for the report. The documentation was fixed in #241; 

Ah OK - I did search, honest! 

> the docs on the web are for the current release, not master. 

OK, slightly awkward from a user perspective (it doesn't have to be a code release) but understandable (you want to keep updates all together). 

> The [OMNI2](https://omniweb.gsfc.nasa.gov/html/overview.html) dataset is indeed hourly; the one-minute set is [high-resolution OMNI](https://omniweb.gsfc.nasa.gov/ow_min.html). The high-res OMNI is of course a much larger data set, so adding SpacePy handling of it would be a substantive effort. The current approach of downloading and processing it all in advance wouldn't be practicable.

Yep, realised that after posting. Going off topic now: I wonder if, instead of pre-processing, the dataset could be processed on request (i.e. by year/month) and then saved for future use. I'm sure it could but would be an effort like you say. Which reminds me to say: thank you to the whole Spacepy team for your useful package.

**Comment by jtniehof** [comment]
The only reason I could find that PR was because I remember writing it, no apologies necessary :)

Agreed that HR OMNI support would need to be more of an on-demand, and there are loose ideas for some sort of on-demand download support which could back that, but nothing concrete right now.

And you're welcome!

**Comment by balarsen** [comment]
@jtniehof Should this be closed based on #241?

**Comment by jtniehof** [comment]
Yes, now that the release is out the web docs are updated. I think that's why I left this open just so people could find it if they were reading the docs pre-release.

I opened #488 for the idea of supporting the one-minute OMNI.
